tests/ginkgo-e2e/scripts/build-airgap

```diff
@@ -85,15 +89,19 @@ for i in k3s-airgap-images-amd64.tar.zst k3s; do
   curl -sfL https://get.k3s.io -o ${OPT_RANCHER}/k3s/install.sh
 
   # Get the airgap deploy script
-  cp ${DEPLOY_AIRGAP_SCRIPT} ${OPT_RANCHER}/k3s
+  cp ${DEPLOY_AIRGAP_SCRIPT} ${UPGRADE_AIRGAP_SCRIPT} ${OPT_RANCHER}/k3s
 done
 
 # Add k3s artifacts to hauler
 ${HAULER_BIN} store add file ${OPT_RANCHER}/k3s
 
 # Pull Kubewarden artifacts using the Hauler manifest
 cd ${OPT_RANCHER}
-${HAULER_BIN} store sync --filename ${HAULER_MANIFEST}
+if [[ "$TEST_TYPE" == "upgrade" ]] ; then
+  ${HAULER_BIN} store sync --filename ${HAULER_MANIFEST_PREV}
```

Comment:
Question: if we are performing the upgrade, shouldn't we sync HAULER_MANIFEST instead of HAULER_MANIFEST_PREV?

Reply:
no because before upgrading, we want to install the previous version of everything so we need to push what we have in the previous latest stable manifest in the registry. And after the installation, we sync the registry with `HAULER_MANIFEST`.